GitUpKit/Extensions/NSView+Embedding.m

```diff
@@ -0,0 +1,37 @@
+//
+//  NSView+Embedding.m
+//  GitUpKit (OSX)
+//
+//  Created by Dmitry Lobanov on 06.04.2020.
+//
+
+#import "NSView+Embedding.h"
+
+@implementation NSView (Embedding)
+- (void)embedView:(NSView *)view {
+  [self.class embedView:view inView:self];
+}
+
++ (void)embedView:(NSView *)view inView:(NSView *)superview {
+  [superview addSubview:view];
+  view.translatesAutoresizingMaskIntoConstraints = NO;
+  
+  if (superview != nil && view != nil) {
+    if (@available(macOS 10.11, *)) {
```

Comment:
nit: @lolgear think about deprecate low mac versions

Reply:
@NikKovIos I’ll take a pass at cleaning this up on a new branch. The original author did enough on this one for me to push it over the finish line.